src/drug_discovery/abfe.py

```diff
@@ -255,6 +255,27 @@ def find_dt_violations(obj, path: list[str]) -> list[str]:
                 f"Found invalid dt values; must be numeric and within range [{min_dt}, {max_dt}]. Offending paths: {paths}"
             ) from None
 
+    def quote(
```

Comment:
I do not think we want separate quote functions. It should be integrated with the run.